src/components/Header/header.js

```diff
@@ -16,30 +24,53 @@ class Header extends Component  {
         };
 
         this.responsiveUtils = new ResponsiveUtils();
+        
     }
     componentDidMount(){
-
+        
         // On Resize and On Orientation Change Handler added to ResponsiveUtils Object
         this.responsiveUtils.__proto__.onResize = () => {
             this.responsiveUtils.viewPortSelector();
             this.setState ({
-                viewport : this.responsiveUtils.viewPort
+                viewport : this.responsiveUtils.viewport
```

Comment:
Make sure viewport spelling in any other files.

Reply:
Fixed